app/views/admin/users/show.html.erb

```diff
@@ -9,3 +7,6 @@
   <%=@user.zip %><br>
   Email: <%=@user.email %><br>
 </section><br>
+<div align="center">
+  <%= link_to "Go Back", :back %>
```

Comment:
We should add more "Go Back" links just for ease of use during refactor

Reply:
Agreed